astra-sim/system/Sys.cc

```diff
@@ -6,8 +6,9 @@ LICENSE file in the root directory of this source tree.
 #include "astra-sim/system/Sys.hh"
 
 #include <iostream>
+#include <cstdlib>
 
-#include "astra-sim/json.hpp"
+#include <json/json.hpp>
```

Comment:
If you are referring to the file in `extern/helper/json/json.hpp`, using <> may not be a good option. Let's talk about this.

Reply:
Yes we can talk about better code conventions in general. fyi: The convention I used throughout is:
- For files within `extern`, I used angle brackets (e.g., `<json/json.hpp>` or `<astra-network-analytical/EventQueue.hh>`
- And for files within the same project, I used quotation marks (e.g., `"astra-sim/Sys.hh"`)